code/modules/wiremod/components/action/soundemitter.dm

```diff
@@ -76,6 +76,6 @@
 	if(!sound_to_play)
 		return
 
-	playsound(src, sound_to_play, volume.input_value, FALSE, frequency = frequency.input_value)
+	playsound(src, sound_to_play, volume.input_value, frequency? TRUE : FALSE, frequency = frequency.input_value)
```

Comment:
```suggestion
	playsound(src, sound_to_play, volume.input_value, frequency != 0, frequency = frequency.input_value)
```

Again, mostly pedantic, but `? TRUE : FALSE` never reads good to me.

Reply:
Couldn't you just leave it as `frequency`?

Reply:
Unless you're trying to do some weird type-hinting at the parameter expecting a boolean value